### DISCUSSION TITLE
https://mnfienen.github.io/talulat/2023/08/13/day12.html

### DISCUSSION THREAD

**Discussion by giscus** [mnfienen/talulat#21] [Comments]
# https://mnfienen.github.io/talulat/2023/08/13/day12.html

Day 12 - LOOOOONG day into Quebec City! miles today:108 total miles: 777 km today: 174 total km: 1252

https://mnfienen.github.io/talulat/2023/08/13/day12.html

**Comment by nanjopas** [comment]
Too happy to have had rain, it was the trigger that allowed me to have a great meeting. Fantastic people with a big smile, only happiness and good times during this 40 minutes under the bridge sheltered from the rain. Mike and Chandra, looking forward to seeing you again and meeting you again, keep having fun and also stretching under the bridges, it's simply magical. Have a safe trip to Halifax

**Reply by mnfienen** [reply]
@nanjopas  - sorry to have missed this comment - we had trouble keeping up with the comments through the phone app. But....so glad you checked in and it was a highlight having out chace meeting under the bridge! We made to Halifax, in the airport on the way home now, and hope the rest of your ride was fun and dry that day!
